src/components/Footer/RenderCompanyInfoSection.jsx

```diff
@@ -37,34 +48,19 @@ const socialLinks = [
  * @returns {React.JSX.Element} The RenderCompanyInfoSection component
  */
 const RenderCompanyInfoSection = ({ isReallySmallScreen }) => (
-  <Stack
-    width={isReallySmallScreen ? 1 : 1 / 5}
-    spacing={2}
-    justifyContent="space-around"
-    alignItems="center"
-  >
+  <Stack width={isReallySmallScreen ? 1 : 1 / 5} justifyContent="space-between" alignItems="center">
     <Box>
-      <Typography color="tertiary.main">PASS LOGO</Typography>
-      <Typography color="#fff">tagline</Typography>
-    </Box>
-    <Box>
-      <Typography color="tertiary.main">Follow Us</Typography>
-      <Stack direction="row" spacing={1}>
+      <Typography color="tertiary.main" variant="h5" mb={1}>
```

Comment:
Now the icons aren't as responsive and have issues at certain screen sizes.

<img width="613" alt="Screenshot 2023-09-11 at 9 11 00 AM" src="https://github.com/codeforpdx/PASS/assets/71395931/40ac419a-63bb-4c47-ba21-28e2865a7c8b">